MoMMI/Modules/CodeHandling/dm.py

```diff
@@ -68,6 +69,10 @@ def __init__(self) -> None:
                     continue
 
                 lines[index] = "\t" + line
+                if line.find('#include "'):
+                    library = line[10:-1]; # Fetch the library name within the ""
+                    libToCopy = libsDir.glob(library+".dm")
```

Comment:
You should sanitize to make sure the path is actually in there and there's no hackery with like `../`